src/v2/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -388,26 +388,17 @@ const renderPricing = (
   }
 }
 
-const renderEstimate = (estimatedPrice, user, mediator, size) => {
-  const justifyContent = size === "xs" ? "flex-start" : "flex-end"
+const renderEstimate = (estimatedPrice, user, mediator) => {
+  let body: JSX.Element
```

Comment:
I _think_ moving the branching to the top of this function is helpful, but I might just be faffing about.